### DISCUSSION TITLE
Export SVG and multiple reaction arrows

### DISCUSSION THREAD

**Discussion by Dom1L** [epam/ketcher#567] [Q&A]
Hi, 

thanks for developing Ketcher! 
It's an extremely convenient drawing tool and I would also like to use it to quickly sketch multistep reactions and export the canvas as an svg. Would this be possible?

In that regard, I noticed that it is not possible to add more than one reaction arrow in a canvas. For multistep reactions this would be extremely important. 

Features such as converting to SMILES would not be necessary for this, it's solely about the awesome drawing capabilities of Ketcher.

Thanks!

**Comment by AndreiMazol** [comment]
Dear @Dom1L ,

thank you for your interest in Ketcher.

The possibility to add any reaction arrows has already been added to  Ketcher (it will be released in scope of release v.2.3)

![image](https://user-images.githubusercontent.com/72735611/120773279-e0c0e100-c529-11eb-8be1-0589f8971c53.png)

Import structure as Image has already been implemented too. This feature has some bugs this moment but all of them  will be fixed soon. 

![image](https://user-images.githubusercontent.com/72735611/120773856-5f1d8300-c52a-11eb-97ff-9424822afc8d.png)

You can create pre-production build to use this functionality right now. Please look  at [insruction](https://github.com/epam/ketcher/blob/master/DEVNOTES.md)

Best regards,
Andrei

**Reply by ly-sniper** [reply]
Dear @AndreiMazol 
`Save As Image` is a nice feature.  
But is there a plan to save '**texts**' and '**structure**' into image?

**Reply by AndreiMazol** [reply]
Hi @hayabusa2009 ,

all elements that can be added on canvas  will be rendered as Image. This moment we have some bottleneck on Indigo side but we plan to deliver this functionality as a part of the nearest release.
